### PR TITLE
Add FOSSA to Drone Pipeline

### PR DESCRIPTION
**Pull request by oxr463**
This adds a fossa analyze and test step to the drone pipeline.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -14,6 +14,26 @@ steps:
   volumes:
   - name: docker
     path: /var/run/docker.sock
+- name: fossa
+  image: registry.suse.com/suse/sle15:15.3.17.8.1
```
Comment (nickgerace) [inline .drone.yml:18]: This will need to be `registry.suse.com/suse/sle15:15.3` as discussed offline.

**Review by nickgerace — CHANGES_REQUESTED** [review]

**Review by nickgerace — APPROVED** [review]

**Comment by nickgerace** [comment]
@StrongMonkey I think we are good here with the image tag fixed. Thoughts?

**Comment by StrongMonkey** [comment]
Looks good to me. Have we run this locally to see if it will succeed? I don't expect this to succeed though.

**Comment by oxr463** [comment]
> Looks good to me. Have we run this locally to see if it will succeed? I don't expect this to succeed though.

I've tested it locally without any issues. Please let me know if you need any assistance.

**Comment by StrongMonkey** [comment]
@oxr463 Thanks, I am worried about breaking the pipeline without knowing how to fix it. Given it has been tested, LGTM!

**Review by StrongMonkey — APPROVED** [review]

**Pull request merged**